data/model/org_mirror.py

```
@@ -244,3 +252,680 @@ def delete_org_mirror_config(org):
+            OrgMirrorConfig.sync_transaction_id == org_mirror_config.sync_transaction_id,
+        )
+
+        updated = query.execute()
+
+    return OrgMirrorConfig.get_by_id(org_mirror_config.id) if updated else None
+
+
+def release_org_mirror_config(
+    org_mirror_config: OrgMirrorConfig,
+    sync_status: OrgMirrorStatus,
+    repos_discovered: int = 0,
```

Ruff (0.14.13)

Unused function argument: `repos_discovered`

(ARG001)